index.js

```diff
@@ -29,6 +30,7 @@ let getExpiresOn = function(session, ttl){
 
 var MysqlStore = function (options) {
     let pool = null
+    let cleanInterval = (!options.clean) ? FIFTEEN_MINUTES : options.clean;
```

Comment:
Why not:
`let cleanInterval = options.clean ? options.clean : FIFTEEN_MINUTES;`